src/contexts/DataContext.js

```diff
@@ -0,0 +1,38 @@
+import { createContext, useState } from 'react';
+
+export const DataContext = createContext();
+
+const DataContextProvider = (props) => {
```

Comment:
To be explicit I would do `({ children })` instead so it's clear from the very beginning what is being passed in.  Then we'd know nothing sneaky happens before in between or after it gets spit out.

Reply:
Not quite sure what you mean here. Are you saying to replace (props) with ({children}), when I tried that it wouldn't render because it says props is not defined.

Reply:
Correct!  Then you need to simplify the below to be `{children}` instead of `{props.children}`